packages/light.js/src/rpc/utils/createRpc.spec.ts

```diff
@@ -6,28 +6,34 @@
 import { timer } from 'rxjs';
 
 import createRpc from './createRpc';
+import { resolveApi } from '../../utils/testHelpers/mockApi';
+import { setApi } from '../../api';
+
+beforeAll(() => {
+  setApi(resolveApi());
+});
 
```

Comment:
can add a test for should memoize based on arguments & api

```js
setApi(resolveApi())
const a = createRpc({})()();
const a2 = createRpc({})()();
exect a === a2
setApi(resolveApi());
const b = createRpc({})()();
expect a !== b
```

```js
setApi(resolveApi())
const a = createRpc({})()('a');
const b = createRpc({})()('b');
expect a !== b
```

same for frequency tests